### PR TITLE
nzbget: use brew binaries for helpers like unrar and 7z

### PR DESCRIPTION
**Pull request by tmm1**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/nzbget.rb [hunk 1]
```diff
@@ -52,6 +52,11 @@ def plist; <<~EOS
     <dict>
       <key>Label</key>
       <string>#{plist_name}</string>
+      <key>EnvironmentVariables</key>
+      <dict>
+        <key>PATH</key>
+        <string>/usr/local/bin/:/usr/bin:$PATH</string>
```
Comment (fxcoudert) [inline Formula/nzbget.rb]: - Isn't `/usr/bin` in `PATH` by default?
- Use `#{HOMEBREW_PREFIX}/bin` for Homebrew path

Comment (tmm1) [reply Formula/nzbget.rb]: I copied this from another formula. I've made the requested changes.

**Comment by tmm1** [comment]
Ping?

**Comment by fxcoudert** [comment]
`$PATH` is not allowed in a plist key, as far as I can tell. The fact that you cannot test the changes probably means they should be tested, by someone with real use case, before suggested as a PR. Closing for now.

**Comment by tmm1** [comment]
I wasn't sure if `$PATH` was valid either, but assumed it was because I copied these changes from a working setup which was using it.

Upon further testing it looks like `$PATH` is indeed invalid, as confirmed by using `ps eww $(pidof nzbget)` to examine the value of the PATH environment variable (h/t https://apple.stackexchange.com/a/140338/235811).

I've updated my branch according, to remove `$PATH` and use the standard macos path instead. Would you mind re-opening this PR and reviewing the changes? Without this change its hard to use nzbget on macOS since it relies on other packages like unrar/7z/par2 for operation.

**Comment by fxcoudert** [comment]
I cannot reopen this one (don't know why… is the branch deleted?). Please open a new PR once you have confirmed locally that it works. Please describe also what is failing without it (messages, etc.).

**Pull request closed without merging**